website/src/fieldsGenerator/dataAccessFields.js

```diff
@@ -8,8 +8,8 @@ export const dataAccessFields = (path, yamlData) => {
     const connections = yamlData.core_lib.connections
     const entities = yamlData.core_lib.entities
     const fields = []
-    const connection = []
-    const entity = []
+    const connection = ['']
```

Comment:
what does this solve?

Reply:
this is for the user to select a empty connection